ansible/roles/postgresql/tasks/backup_plain.yml

```diff
@@ -1,4 +1,14 @@
 ---
+- name: Copy replication credentials
+  become: yes
+  template:
+    src: "backup/plain/pgpass.j2"
+    dest: "{{ postgresql_dir_original_path }}/.pgpass"
```

Comment:
@javierwilson 
I tried again and it turns out the issue I mentioned in https://github.com/dimagi/commcarehq-ansible/pull/1025/files#r154541574 actually happens here, not there. When it renders the template `backup/plain/pgpass.j2` (below in this PR) it fails with the message "AnsibleUndefinedVariable: 'hot_standby_master' is undefined".

I'm testing off (a very-slightly modified version of) the [dev](https://github.com/dimagi/commcarehq-ansible/tree/master/ansible/vars/dev) variables, with nothing in the `[pg_standby]` group and no `hot_standby_master` defined anywhere, very much like [staging](https://github.com/dimagi/commcare-hq-deploy/blob/master/fab/inventory/staging#L51). Was this change maybe made assuming that this would only ever be run on a pg_standby machine, with `hot_standby_master` always defined?

Reply:
Was able to fix this by just adding `when: is_pg_standby` here. Sound good if I PR that?

Reply:
yes, i think so... that file should only go on backup server (standby) i'll apply the same fix for https://github.com/dimagi/commcarehq-ansible/pull/1104